LayoutTests/compositing/hidpi-image-backing-store-scaled.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<script>
+
+if (window.testRunner) {
+  testRunner.dumpAsText();
+  testRunner.waitUntilDone();
+}
+
+
+function doTest() {
+  document.getElementById('layers').innerText = internals.layerTreeAsText(document, internals.LAYER_TREE_INCLUDES_DEVICE_SCALE);
+  testRunner.notifyDone();
+}
+
+function loadImage() {
+  let img = document.getElementById('image');
+  img.onload = doTest;
+  img.src = "resources/simple_image.png";
+
+}
+
+window.addEventListener('load', loadImage, false);
+</script>
+</head>
+
+<body>
+<p> Tests whether a compositing layer containing only an image uses a standard DPI backing store.</p>
+<img id="image" style="transform: translateZ(0px); width: 200px; height: 150px"></img>
```

Comment:
On macOS this will fall into our "simple image" optimization where we put a CGImageRef onto a CALayer as layer contents (`setContentsToImage()`) in which case I'm not sure the layer device scale does anything. But the GraphicsLayer dump won't reflect what CoreAnimation is doing.

Reply:
The tests are still valid iOS test, however, where we don't have the "simple image" optimization.

Reply:
Indeed! I figured it was best to just run them on both platforms though, and have different expectations accordingly.